JS-condition/js-conditions-and-booleans_teenager-check/js/logType.js

```diff
@@ -0,0 +1,50 @@
+let data;
+
+let typeOfData = typeof data;
+
+let message = "";
+
+switch (typeOfData) {
+  case "undefined":
+    message = "Undefine!";
+    break;
+
+  case "string":
+    message = "String!";
+    break;
+  case "bigInt":
+    message = "BigInt!";
+    break;
+  case "function":
+    message = "Function!";
+    break;
+  case "boolean":
+    message = "boolean!";
+    break;
+
+  //number & NaN
+
+  case "number":
+    if (Number.isNaN(data)) {
+      message = "Not a Number";
+      break;
+    }
+    message = "Number!";
+    break;
+
+  case "object":
+    if (Array.isArray(data)) {
+      message = "Array";
+      break;
+    } else if (data == null) {
```

Comment:
A triple equal `===` would make it more explicit and safe that you're comparing with the exact value of `null`. In this case if `data = undefined`, `data == null` would evaluate to `true` which is probably not what you want. For more info on this: I'd suggest reading up on [equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality) vs [strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality).